bigquery/setup.py

```diff
@@ -58,6 +58,10 @@
     'requests >= 2.18.0',
 ]
 
+EXTRAS_REQUIREMENTS = {
+    'pandas': ['pandas >= 0.3.0'],
```

Comment:
You sure you don't want a later release? This one is from 2011 and the lastest one is 0.21.0

Reply:
This is the earliest version of pandas that will work. Should I use a different metric for choosing a version?

Reply:
@jonparrott WDYT?